File: HigherLowerGame/HigherLower.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HigherLowerGame
{
    class HigherLower
    {
        static void Main(string[] args)
        {
            var rand = new Random();
            int number = rand.Next(101);
            int count = 0;

            Console.Write("Pick a number between 1-100: ");
            int guess = Convert.ToInt32(Console.ReadLine());

            while (number != guess)
            {
                if (number > guess)
                {
                    Console.Write("Higher.Try again: ");
                    count++;
                    guess = Convert.ToInt32(Console.ReadLine());
                }
                else if (number < guess)
                {
                    Console.Write("Lower.Try again: ");
                    count++;
                    guess = Convert.ToInt32(Console.ReadLine());
                }
            }
            if (number == guess)
            {
                Console.WriteLine("Correct! You guessed in your {0}. try.", count);
            }

            Console.ReadLine();
        }
    }
}

File: higher_lower.py
import random as rnd

number = rnd.randint(1,100)
guess = int(input("Guess the number between 1-100: "))

count = 0
while number != guess:
    if number < guess:
        count += 1
        guess = int(input("Lower. Try again: "))
    elif number > guess:
        count += 1
        guess = int(input("Higher. Try again: "))
        
if number == guess:
    print("Congratulations. You guessed in your {}. try ".format(count+1))

